src/assets/i18n/en.json

```diff
@@ -1,4 +1,6 @@
 {
+	"Managed by Arlas": "This field will be automatically filled by Arlas",
```

Comment:
I propose : 
In case of displaying date fields on the x-axis, the unit is automatically managed and displayed by ARLAS.